tokenizer.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
Nice use of the shebang!  Although I think this would try to run a python from the root directory.  I think what you need is `#!./usr/bin/python`

They both actually work, however one is running a python outside your virtual environment, the second insures you run the one in your local env directory.